### PR TITLE
[Tommy Collison] Auth-i

### PR DESCRIPTION
**Pull request by tommycollison**
@johnoro 🔏 🔐

### CODE REVIEW

**Comment by tommycollison** [comment]
MVP complete!
- [x] POST: /api/register
- [x] POST: /api/login
- [x] GET | /api/users

File: index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        } else {
+            res.status(404).json(`You shall not pass!`)
+        }
+    })
+    .catch(err => {res.status(500).send(err)})
+})
+
+
+server.post('/api/login', (req, res) => {
+})
```
Comment (johnoro) [inline index.js]: It looks like this piece of code is unnecessary! You already have your post up above.

File: index.js [hunk 2]
```diff
@@ -47,8 +65,15 @@ server.post('/api/login', (req, res) => {
...
+            res.status(500).send(`failed to log you out`)
+        } else {
+            res.status(201).send('successfully logged you out')
+        }
+    })
 
-server.post('/api/login', (req, res) => {
 })
 
 server.listen(2525, () => console.log(`I'm alive!`))
```
Comment (johnoro) [inline index.js]: Typically we want a port variable, i.e.:
```javascript
const PORT = 2525;
server.listen(PORT, () => console.log(`I'm alive on port ${port}!`));
```

File: database/dbHelpers.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+get: (users) => {
+    return db('users').select('id', 'username')
+},
+
+insert: (user) => {
+    return db('users').insert(user);
+},
+
+findByUsername: (username) => {
+    return db('users').where('username', username);
```
Comment (johnoro) [inline database/dbHelpers.js]: You could use an object here to make the argument for your `where` a tad shorter:
```javascript
return db('users').where({ username });
```

**Pull request closed without merging**